wandb/sdk/wandb_run.py

```diff
@@ -147,14 +147,33 @@ class RunStatusChecker(object):
     For now, we just use this to figure out if the user has requested a stop.
     """
 
-    def __init__(self, interface: BackendSender, polling_interval: int = 15) -> None:
+    def __init__(self, interface: BackendSender, stop_polling_interval: int = 15, retry_polling_interval: int = 1) -> None:
```

Comment:
polling interval 1s is pretty arbitrary, i think we want something fairly frequent as error printing should feel close as possible to realtime